### PR TITLE
fix: Since Xcode 9.3, IWDP cannot connect to the IOS simulator via tcp

### PR DESCRIPTION
**Pull request by aloneszjl**

### CODE REVIEW

File: lib/remote-debugger.js [hunk 1]
```diff
@@ -10,6 +11,42 @@ const WebSocketClient = require('./client');
 
 const SIMULATOR = 'SIMULATOR';
 
+// https://regex101.com/r/MEL55t/1
+const WEBINSPECTOR_SOCKET_REGEXP = /\s+(\S+com\.apple\.webinspectord_sim\.socket)/;
+
+const getWebInspectorSocket = deviceId =>
+  new Promise(resolve => {
```
Comment (xudafeng) [inline lib/remote-debugger.js:18]: return new Promise?

File: lib/remote-debugger.js [hunk 2]
```diff
@@ -10,6 +11,42 @@ const WebSocketClient = require('./client');
...
+    //   launchd_s 69760 isaac    8u  unix 0x57aa4fceea39598b      0t0      /private/tmp/com.apple.launchd.2j5k1TMh6i/com.apple.webinspectord_sim.socket
+    //   launchd_s 69760 isaac    9u  unix 0x57aa4fceea394c43      0t0      /private/tmp/com.apple.launchd.4zm9JO9KEs/com.apple.testmanagerd.unix-domain.socket
+    //   launchd_s 69760 isaac   10u  unix 0x57aa4fceea395f03      0t0      /private/tmp/com.apple.launchd.2B2u8CkN8S/Listeners
+    //   launchd_s 69760 isaac   11u  unix 0x57aa4fceea39598b      0t0      /private/tmp/com.apple.launchd.2j5k1TMh6i/com.apple.webinspectord_sim.socket
+    //   launchd_s 69760 isaac   12u  unix 0x57aa4fceea394c43      0t0      /private/tmp/com.apple.launchd.4zm9JO9KEs/com.apple.testmanagerd.unix-domain.socket
+    // these _appear_ to always be grouped together (so, the records for the particular sim are all in a group, before the next sim, etc.)
+    // so starting from the correct UDID, we ought to be able to pull the next record with `com.apple.webinspectord_sim.socket` to get the correct socket
+    exec('lsof -aUc launchd_sim', (err, stdout) => {
+      if(err) {
```
Comment (xudafeng) [inline lib/remote-debugger.js]: lint style

```
if( => if (
```

File: lib/remote-debugger.js [hunk 3]
```diff
@@ -10,6 +11,42 @@ const WebSocketClient = require('./client');
...
+        resolve(null)
```
Comment (xudafeng) [inline lib/remote-debugger.js]: lint style

;

File: lib/remote-debugger.js (hunk 1 above)
Comment (aloneszjl) [reply lib/remote-debugger.js:18]: yeah

File: lib/remote-debugger.js (hunk 2 above)
Comment (aloneszjl) [reply lib/remote-debugger.js]: fixed!

File: lib/remote-debugger.js (hunk 3 above)
Comment (aloneszjl) [reply lib/remote-debugger.js]: fixed!

**Pull request merged**